qa/integration-tests/src/test/java/io/camunda/it/rdbms/db/usertask/UserTaskIT.java

```diff
@@ -208,6 +209,43 @@ public void shouldFindUserTaskByVariableName(final CamundaRdbmsTestApplication t
     assertUserTaskEntity(searchResult.hits().getFirst(), randomizedUserTask);
   }
 
+  @TestTemplate
+  void shouldFindUserTaskByLocalVariableName(final CamundaRdbmsTestApplication testApplication) {
```

Comment:
I am aware of this test is failing because this:

`testApplication.getRdbmsService();` is returning null, the test is working locally. I am investigating the reason